kolibri/core/assets/test/api-resource.js

```diff
@@ -34,25 +34,25 @@ describe('ResourceManager', function () {
   });
   describe('registerResource method', function () {
     it('should throw a TypeError if no className is passed', function () {
```

Comment:
Could you update the descriptions in the `it` clauses, now that we are no longer asserting the type of error thrown?